Datadog/Datadog.xcodeproj/xcshareddata/xcschemes/Datadog iOS.xcscheme

```diff
@@ -121,6 +121,26 @@
             value = "1"
             isEnabled = "YES">
          </EnvironmentVariable>
+         <EnvironmentVariable
+            key = "DD_APPLICATION_KEY"
+            value = "$(DD_SDK_SWIFT_TESTING_APPLICATION_KEY)"
+            isEnabled = "YES">
+         </EnvironmentVariable>
+         <EnvironmentVariable
+            key = "DD_CIVISIBILITY_GIT_UPLOAD_ENABLED"
+            value = "1"
+            isEnabled = "YES">
+         </EnvironmentVariable>
+         <EnvironmentVariable
+            key = "DD_CIVISIBILITY_ITR_ENABLED"
+            value = "1"
+            isEnabled = "YES">
+         </EnvironmentVariable>
+         <EnvironmentVariable
+            key = "DD_CIVISIBILITY_EXCLUDED_BRANCHES"
+            value = "develop"
```

Comment:
Is it possible to exclude multiple branches using wildcard `*`? I think we may need to exclude `release/*` and `hotfix/*` as well. This is to elevate safety and run all tests when we prepare a new release. WDYT @maxep @nachoBonafonte ?

Reply:
Not currently, but I can add it. Take note also that this functionality (excluding branches) wont use an environment variable in the tracer but will be handled directly in Datadog and the tracer will ask the backend for the configuration